From d79f9e1c33988ee75f2c4026d449c95eaee4bb75 Mon Sep 17 00:00:00 2001
From: Abhishek <apatgaonkar@habana.ai>
Date: Tue, 16 Nov 2021 11:30:59 +0200
Subject: [PATCH] [SW-0] gaudi2: configure pci pll

For fw loader agent to be loaded some basic
plls need to be configured. In case of preboot
PCI pll configuration is added which is
necessary for agent fit copy.

Change-Id: I51c33db7db987958431d948571bee8017c4e97d1
---

diff --git a/arch/gaudi2/common/asic.c b/arch/gaudi2/common/asic.c
index cab2480..97b9fc5 100644
--- a/arch/gaudi2/common/asic.c
+++ b/arch/gaudi2/common/asic.c
@@ -45,6 +45,16 @@
 
 static struct hl_pll_cfg gaudi2_pll_cfg[] = {
 	{
+		.name = "PCI_PLL",
+		.base = mmDCORE0_PCI_PLL_CTRL_BASE,
+		.nr = 1, .nf = 60, .od = 3, .nb = 2,	/* 1600 */
+		.div_factor = { -1, -1, 4, 4 },
+		.div_sel = { HL_PLL_PLL_CLK,
+			     HL_PLL_PLL_CLK,
+			     HL_PLL_DIV_PLL,
+			     HL_PLL_DIV_PLL
+		},
+	}, {
 		.name = "DCORE0_IF",
 		.base = mmDCORE0_XBAR_IF_PLL_CTRL_BASE,
 		.nr = 1, .nf = 64, .od = 2, .nb = 1,	/* 1600 */
